libmscore/harmony.cpp

```diff
@@ -707,6 +708,8 @@ void Harmony::startEdit(EditData& ed)
       {
       if (!textList.empty())
             setXmlText(harmonyName());
+
+      layoutInvalid = false;
```

Comment:
I miss the place where we set `layoutInvalid` to `true`. Constructor is not taken into account.

Reply:
setXmlText() set layoutInvalid to true